src/main/java/com/notably/logic/commands/suggestion/DeleteSuggestionCommand.java

```diff
@@ -98,8 +99,9 @@ private void getChildDfs(BlockTreeItem curr, List<String> components,
         return possiblePaths.stream()
                 .map(path -> {
                     String displayText = path.getStringRepresentation();
+                    String updatedInput = (model.getInput()).replace(title, displayText);
```

Comment:
Nit: Actually, the `(model.getInput())` part is rather unnecessary. You can write it as `model.getInput().replace()` and things would still work fine